diff --git a/lib/client.js b/lib/client.js
index 0e7372f..d52a8d4 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -13781,7 +13781,7 @@ window.__ModuleLoader__.load({
 			"access.confirm.acknowledge": "我已了解风险，并愿意继续",
 			"access.confirm.cancel": "取消",
 			"access.confirm.enable": "启用完全权限",
-			"hero.headline": "探索未至之境",
+			"hero.headline": "金石易服",
 			"hero.preview": "预览版",
 			"hero.chooseWorkspace": "选择工作区",
 			"session.hierarchy": "会话层级",
@@ -13943,7 +13943,7 @@ window.__ModuleLoader__.load({
 			"access.confirm.acknowledge": "I understand the risks and want to continue",
 			"access.confirm.cancel": "Cancel",
 			"access.confirm.enable": "Enable Full access",
-			"hero.headline": "Into the Unknown",
+			"hero.headline": "金石易服",
 			"hero.preview": "Preview",
 			"hero.chooseWorkspace": "Choose workspace",
 			"session.hierarchy": "Session hierarchy",
